ADMVEDU/Protocols/DetailsDelegate.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  MediaCellDelegate.swift
+//  ADMVEDU
+//
+//  Created by Bobr, Andrey on 6.05.21.
+//
+
+import Foundation
+
+protocol DetailsDelegate: AnyObject {
```

Comment:
1. DetailsViewControllerDelegate? let's make it more clear
2. https://github.com/raywenderlich/swift-style-guide/blob/master/README.markdown#delegates